obsreport/obsreport_exporter.go

```diff
@@ -63,6 +63,11 @@ func (eor *Exporter) EndTracesExportOp(ctx context.Context, numSpans int, err er
 	endSpan(ctx, err, numSent, numFailedToSend, obsmetrics.SentSpansKey, obsmetrics.FailedToSendSpansKey)
 }
 
+// RecordTracesEnqueueFailure records number of spans that failed to be added to the sending queue.
+func (eor *Exporter) RecordTracesEnqueueFailure(ctx context.Context, numSpans int) {
+	_ = stats.RecordWithTags(ctx, eor.mutators, obsmetrics.ExporterFailedToEnqueueSpans.M(int64(numSpans)))
+}
```

Comment:
I want to think a bit more about this API, do you know any simple way to make this "private API" (internal) for the moment until we better understand the need?

Reply:
Cannot think of any other way to arrange this without code duplication :(

Reply:
Temporarily migrated to make it private with some code duplication